src/huggingface_hub/commands/delete_cache.py

```diff
@@ -275,7 +318,11 @@ def _manual_review_no_tui(hf_cache_info: HFCacheInfo, preselected: List[str]) ->
     os.close(fd)
 
     lines = []
-    for repo in sorted(hf_cache_info.repos, key=_repo_sorting_order):
+    sorted_repos = sorted(
+        hf_cache_info.repos,
+        key=lambda repo: -repo.size_on_disk if sort_by == "size" else 1,
```

Comment:
Can you move the sorting logic defined above in the TUI part to a separate method so that it can be reused here? So that the "no tui" path can also support all sorting options.